test/test_filters.rb

```diff
@@ -475,7 +481,11 @@ def to_liquid
               g["items"].is_a?(Array),
               "The list of grouped items for '' is not an Array."
             )
-            assert_equal 13, g["items"].size
+            if Jekyll::Utils::Platforms.windows?
```

Comment:
This should be a unique test.  This kind of tight coupling is terrifying (not your fault.)